lib/constants.dart

```diff
@@ -135,6 +135,9 @@ class Buttons {
 
   // Android only
   static const editAnnotationToolbarButton = 'editAnnotationToolButton';
+  static const saveReducedCopyButton = 'saveReducedCopyButton';
```

Comment:
how come there are 5 on top and only 3 here?

Reply:
The other two were already added as constants because they were implemented for iOS